File: 1012.c
#include <stdio.h>
#include <stdlib.h>

int main() 
{
    double A,B,C,Y,triangulo,circulo,trapezio,quadrado,retangulo;
          scanf("%lf %lf %lf",&A,&B,&C);
    double pi = 3.14159;
    triangulo = A*C/2;
    circulo = pi*C*C;
    Y=A+B;
    trapezio = Y*C/2;
    quadrado = B*B;
    retangulo = A*B;
    printf("TRIANGULO: %.3lf\nCIRCULO: %.3lf\nTRAPEZIO: %.3lf\nQUADRADO: %.3lf\nRETANGULO: %.3lf\n",triangulo,circulo,trapezio,quadrado,retangulo);
    return 0;
}

File: Kotlin/1012.kt
fun main(args: Array<String>) {
  var A:Double = readLine()!!.toDouble()
  var B:Double = readLine()!!.toDouble()
  var C:Double = readLine()!!.toDouble()
  println("TRIANGULO: %.3f".format((A*C)/2))
  println("CIRCULO: %.3f".format(3.14159 * C * C))
  println("TRAPEZIO: %.3f".format((A+B)/2 * C))
  println("QUADRADO: %.3f".format(B * B))
  println("RETANGULO: %.3f".format(A * B ))
} 